Packs/CommonScripts/Scripts/CommandLineAnalysis/CommandLineAnalysis_test.py

```diff
@@ -116,9 +114,24 @@ def test_check_suspicious_macos_applescript_commands():
     assert "tell application finder" in matches["infostealer_characteristics"][0]
 
 
-# Test analyze_command_line
+def test_custom_patterns_score():
+    """Test that custom patterns are properly scored in command line analysis."""
+
+    command = 'cmd "test"'
+    result = analyze_command_line(command, custom_patterns=["test"])
+    assert result["score"] == 21
```

Comment:
Can you please add to the tests also the path for findings + custom pattern? 
`CommandLineAnalysis.findings.original.[1]`
<img width="420" alt="image" src="https://github.com/user-attachments/assets/3f40c8ef-beab-462b-866c-b2ee9bf48b84" />

Reply:
Done